spec/index.html

```diff
@@ -7410,6 +7410,110 @@ <h5>TZ</h5>
             </div>
           </section>
         </section>
+
+        <!-- QT -->
+        <section id="func-quoted-triples">
+          <h4>Functions on Quoted Triples</h4>
+          <section id="func-triple">
+            <h5>TRIPLE</h5>
+            <pre class="prototype nohighlight">
+              <span class="return">quoted triple</span>  <span class="operator">TRIPLE</span> (<span class="type RDFterm">RDF term</span> <span class="name">subj</span>, <span class="type RDFterm">RDF term</span> <span class="name">pred</span>, <span class="type RDFterm">RDF term</span> <span
+class="name">obj</span>)
+            </pre>
+            <pre class="query nohighlight">
+                  <span class="operator">&lt;&lt;</span> term1 term2 term3 <span class="operator">&gt;&gt;</span>
```

Comment:
Can we here also use `subj`, `pred`, and `obj`, for consistency with the `TRIPLE`-form above?

(also applies to the paragraph below)

Reply:
I agree. It should be consistent. I assume that the reason why Andy used `term1` ... `term3` here (instead of `subj` ... `obj`) is to make it more obvious that these three subexpressions are meant to represent RDF terms. Following that rationale, and still get consistency, it is better to replace `subj` ... `obj` in the signature of the `TRIPLE` function above.

Reply:
Done.

I'll also added that `<<>>` expression syntax is has a restriction to arguments that are variables or constants, whereas `TRIPLE` can take arbitrary expressions.